### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by adityaraute**
# Pull Request

## Description:

 
An attempt to improve the UI of the page.

## Related Issue:

Is not related to any issue.

This is how the page looks now in my browser. Not the best, I admit. Will try to improve gradually. 
screencapture-localhost-4200-2021-01-30-03_13_19

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<h1>Timezone Calculator</h1>
-<p>Enter your event's start and end date + time in the form, then click "Display Times" to generate a chart of timezones!</p>
+<h1 class="header">Timezone Calculator</h1>
+<p id="info">Enter your event's start and end date + time in the form<br />Click "Display Times" to generate a chart of timezones!</p>
```
Comment (nhcarrigan) [inline src/app/app.component.html:2]: We shouldn't be using `br` tags to force spacing. This is an accessibility issue.

File: src/app/app.component.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ text-align: center;
+ line-height: 10rem;
+ font-size: 3rem ;
+}
+
+#info{
+    font-size: larger;
+    line-height: 35px;
+    margin: 4rem auto 2rem auto;
+}
```
Comment (nhcarrigan) [inline src/app/app.component.css:16]: EOF new line here please 🙂

File: src/app/form/form.component.css [hunk 3]
```diff
@@ -1,4 +1,38 @@
...
+  margin: 1rem;
+  float: left;
+}
+/* 
+input, select{
+  float: right;
+  vertical-align: middle;
+  margin-top: 1.5rem;
+  
+} */
```
Comment (nhcarrigan) [inline src/app/form/form.component.css:38]: Here too

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Some minor details I see at first glance (will try to pull down and test later this evening)

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

I may end up changing out the yellow colour later on, possibly.

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by adityaraute** [comment]
Glad you liked the changes. But I am still not satisfied. What more can be done to modernize the design?

**Comment by Vyvy-vi** [comment]
Screenshot 2021-01-30 at 6 10 16 PM
I feel that there's too much gap between timezone and calculator when opened in smaller windows

**Comment by adityaraute** [comment]
Ahh yes, true. The line-height attribute caused this. How else can I center text in the div vertically?

**Pull request merged**